### DISCUSSION TITLE
Pytorch 文档翻译【持续迭代 - 2.x 最新版本】

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/pytorch-doc-zh#625] [Announcements]
## 小技巧

- 可以直接 Copy 1.4 API
    - https://pytorch.apachecn.org/1.4/74/
- 然后对里面的内容进行微调

## 状态表示

✅ 表示已经完成
❌ 表示无人认领
‼️ 表示正在进行

> ❌ ["Community"]()

*   [PyTorch Governance | Build + CI](https://pytorch.org/docs/stable/community/build_ci_governance.html)
*   [PyTorch Contribution Guide](https://pytorch.org/docs/stable/community/contribution_guide.html)
*   [PyTorch Design Philosophy](https://pytorch.org/docs/stable/community/design.html)
*   [PyTorch Governance | Mechanics](https://pytorch.org/docs/stable/community/governance.html)
*   [PyTorch Governance | Maintainers](https://pytorch.org/docs/stable/community/persons_of_interest.html)

> ‼️ ["Developer Notes"]()

*   [CUDA Automatic Mixed Precision examples](https://pytorch.org/docs/stable/notes/amp_examples.html)
*   [Autograd mechanics](https://pytorch.org/docs/stable/notes/autograd.html)
*   [Broadcasting semantics](https://pytorch.org/docs/stable/notes/broadcasting.html)
*   [CPU threading and TorchScript inference](https://pytorch.org/docs/stable/notes/cpu_threading_torchscript_inference.html)
*   [CUDA semantics](https://pytorch.org/docs/stable/notes/cuda.html)
*   [Distributed Data Parallel](https://pytorch.org/docs/stable/notes/ddp.html)
*   [Extending PyTorch](https://pytorch.org/docs/stable/notes/extending.html)
*   [Extending torch.func with autograd.Function](https://pytorch.org/docs/stable/notes/extending.func.html)
*   [Frequently Asked Questions](https://pytorch.org/docs/stable/notes/faq.html)
*   [Gradcheck mechanics](https://pytorch.org/docs/stable/notes/gradcheck.html)
*   [HIP (ROCm) semantics](https://pytorch.org/docs/stable/notes/hip.html)
*   [Features for large-scale deployments](https://pytorch.org/docs/stable/notes/large_scale_deployments.html)
*   [Modules](https://pytorch.org/docs/stable/notes/modules.html)
*   [MPS backend](https://pytorch.org/docs/stable/notes/mps.html)
*   [Multiprocessing best practices](https://pytorch.org/docs/stable/notes/multiprocessing.html)
*   [Numerical accuracy](https://pytorch.org/docs/stable/notes/numerical_accuracy.html)
*   [Reproducibility](https://pytorch.org/docs/stable/notes/randomness.html)
*   [Serialization semantics](https://pytorch.org/docs/stable/notes/serialization.html)
*   [Windows FAQ](https://pytorch.org/docs/stable/notes/windows.html)

> ❌ ["torch.compile"]()

*   [TorchDynamo Overview](https://pytorch.org/docs/stable/dynamo/index.html)
*   [Installing TorchDynamo](https://pytorch.org/docs/stable/dynamo/installation.html)
*   [Getting Started](https://pytorch.org/docs/stable/dynamo/get-started.html)
*   [Guards Overview](https://pytorch.org/docs/stable/dynamo/guards-overview.html)
*   [Custom Backends](https://pytorch.org/docs/stable/dynamo/custom-backends.html)
*   [TorchDynamo Deeper Dive](https://pytorch.org/docs/stable/dynamo/deep-dive.html)
*   [TorchDynamo Troubleshooting](https://pytorch.org/docs/stable/dynamo/troubleshooting.html)
*   [Frequently Asked Questions](https://pytorch.org/docs/stable/dynamo/faq.html)
*   [IRs](https://pytorch.org/docs/stable/ir.html)

> ❌ ["Language Bindings"]()

*   [C++](https://pytorch.org/docs/stable/cpp_index.html)
*   [Javadoc](https://pytorch.org/javadoc/)
*   [torch::deploy](https://pytorch.org/docs/stable/deploy.html)

> ❌ ["Python API"]()

*   ‼️ [torch](https://pytorch.org/docs/stable/torch.html)
*   [torch.nn](https://pytorch.org/docs/stable/nn.html)
*   [torch.nn.functional](https://pytorch.org/docs/stable/nn.functional.html)
*   [torch.Tensor](https://pytorch.org/docs/stable/tensors.html)
*   [Tensor Attributes](https://pytorch.org/docs/stable/tensor_attributes.html)
*   [Tensor Views](https://pytorch.org/docs/stable/tensor_view.html)
*   [torch.amp](https://pytorch.org/docs/stable/amp.html)
*   [torch.autograd](https://pytorch.org/docs/stable/autograd.html)
*   [torch.library](https://pytorch.org/docs/stable/library.html)
*   ‼️ [torch.cuda](https://pytorch.org/docs/stable/cuda.html)
*   [torch.mps](https://pytorch.org/docs/stable/mps.html)
*   [torch.backends](https://pytorch.org/docs/stable/backends.html)
*   ‼️ [torch.distributed](https://pytorch.org/docs/stable/distributed.html)
*   [torch.distributed.algorithms.join](https://pytorch.org/docs/stable/distributed.algorithms.join.html)
*   [torch.distributed.elastic](https://pytorch.org/docs/stable/distributed.elastic.html)
*   [torch.distributed.fsdp](https://pytorch.org/docs/stable/fsdp.html)
*   [torch.distributed.optim](https://pytorch.org/docs/stable/distributed.optim.html)
*   [torch.distributed.tensor.parallel](https://pytorch.org/docs/stable/distributed.tensor.parallel.html)
*   [torch.distributed.checkpoint](https://pytorch.org/docs/stable/distributed.checkpoint.html)
*   [torch.distributions](https://pytorch.org/docs/stable/distributions.html)
*   [torch.\_dynamo](https://pytorch.org/docs/stable/_dynamo.html)
*   [torch.fft](https://pytorch.org/docs/stable/fft.html)
*   [torch.func](https://pytorch.org/docs/stable/func.html)
*   [torch.futures](https://pytorch.org/docs/stable/futures.html)
*   [torch.fx](https://pytorch.org/docs/stable/fx.html)
*   [torch.hub](https://pytorch.org/docs/stable/hub.html)
*   [torch.jit](https://pytorch.org/docs/stable/jit.html)
*   [torch.linalg](https://pytorch.org/docs/stable/linalg.html)
*   [torch.monitor](https://pytorch.org/docs/stable/monitor.html)
*   [torch.signal](https://pytorch.org/docs/stable/signal.html)
*   [torch.special](https://pytorch.org/docs/stable/special.html)
*   [torch.overrides](https://pytorch.org/docs/stable/torch.overrides.html)
*   [torch.package](https://pytorch.org/docs/stable/package.html)
*   [torch.profiler](https://pytorch.org/docs/stable/profiler.html)
*   [torch.nn.init](https://pytorch.org/docs/stable/nn.init.html)
*   [torch.onnx](https://pytorch.org/docs/stable/onnx.html)
*   [torch.onnx diagnostics](https://pytorch.org/docs/stable/onnx_diagnostics.html)
*   [torch.optim](https://pytorch.org/docs/stable/optim.html)
*   [Complex Numbers](https://pytorch.org/docs/stable/complex_numbers.html)
*   [DDP Communication Hooks](https://pytorch.org/docs/stable/ddp_comm_hooks.html)
*   [Pipeline Parallelism](https://pytorch.org/docs/stable/pipeline.html)
*   [Quantization](https://pytorch.org/docs/stable/quantization.html)
*   [Distributed RPC Framework](https://pytorch.org/docs/stable/rpc.html)
*   [torch.random](https://pytorch.org/docs/stable/random.html)
*   [torch.masked](https://pytorch.org/docs/stable/masked.html)
*   [torch.nested](https://pytorch.org/docs/stable/nested.html)
*   [torch.sparse](https://pytorch.org/docs/stable/sparse.html)
*   [torch.Storage](https://pytorch.org/docs/stable/storage.html)
*   [torch.testing](https://pytorch.org/docs/stable/testing.html)
*   [torch.utils.benchmark](https://pytorch.org/docs/stable/benchmark_utils.html)
*   [torch.utils.bottleneck](https://pytorch.org/docs/stable/bottleneck.html)
*   [torch.utils.checkpoint](https://pytorch.org/docs/stable/checkpoint.html)
*   [torch.utils.cpp\_extension](https://pytorch.org/docs/stable/cpp_extension.html)
*   [torch.utils.data](https://pytorch.org/docs/stable/data.html)
*   [torch.utils.jit](https://pytorch.org/docs/stable/jit_utils.html)
*   [torch.utils.dlpack](https://pytorch.org/docs/stable/dlpack.html)
*   [torch.utils.mobile\_optimizer](https://pytorch.org/docs/stable/mobile_optimizer.html)
*   [torch.utils.model\_zoo](https://pytorch.org/docs/stable/model_zoo.html)
*   [torch.utils.tensorboard](https://pytorch.org/docs/stable/tensorboard.html)
*   [Type Info](https://pytorch.org/docs/stable/type_info.html)
*   [Named Tensors](https://pytorch.org/docs/stable/named_tensor.html)
*   [Named Tensors operator coverage](https://pytorch.org/docs/stable/name_inference.html)
*   [torch.\_\_config\_\_](https://pytorch.org/docs/stable/config_mod.html)

> ❌ ["Libraries"]()

*   [torchaudio](https://pytorch.org/audio/stable)
*   [TorchData](https://pytorch.org/data)
*   [TorchRec](https://pytorch.org/torchrec)
*   [TorchServe](https://pytorch.org/serve)
*   [torchtext](https://pytorch.org/text/stable)
*   [torchvision](https://pytorch.org/vision/stable)
*   [PyTorch on XLA Devices](https://pytorch.org/xla/)

**Comment by jiangzhonglian** [comment]
各位好兄弟，我看到了，有一些内容太麻烦就不用翻译。

表格里面的太麻烦就别翻译，只翻译表格外面的说明就行
例如： 只翻译1的位置，2的位置不用翻译


![](https://user-images.githubusercontent.com/9199175/269597695-307cd940-d998-4b8f-9404-8c2cf5a67b21.png)
